CHANGELOG.md

```diff
@@ -0,0 +1,10 @@
+# Ding Pretix Changelog
+
+### 1.0.1
+* EBILL-92: Change pre-sale to start immediately.
+* EBILL-92: Fixed bug where unlimited participant was set to 0 in pretix. 
+* EBILL-92: Removed "ticket type" field. 
+* EBILL-92: Changed behavior of updates so that "price" and "capacity" are no longer updated in pretix.
```

Comment:
also changed behavior, so that alle items with admissionticket = true are updated on insert.